Pester.psm1

```diff
@@ -227,7 +232,31 @@ function Add-AssertionOperator
 
     Add-AssertionDynamicParameterSet -AssertionEntry $entry
 }
+function Test-AssertionOperatorIsDuplicate
+{
+    param (
+        [psobject] $Operator
+    )
+
+    $existing = $script:AssertionOperators[$Operator.Name]
+
+    # Test SupportsArrayObject and scriptblock (as string) are identical
+    $isDuplicate = $existing -and
+        $Operator.SupportsArrayInput -eq $existing.SupportsArrayInput -and
+        $Operator.Test.ToString() -eq $existing.Test.ToString()
 
+    # Test Aliases
+    if ($Operator.Aliases) {
```

Comment:
The problem is here.  Should be `$Operator.Alias`.  Having the wrong property name is causing strict mode failures.

Reply:
🤦‍♂️  Oops. Good catch.